src/dot11/dot11_base.cpp

```diff
@@ -164,6 +164,10 @@ void Dot11::power_mgmt(small_uint<1> new_value) {
     header_.control.power_mgmt = (new_value)? 1 : 0;
 }
 
+void Dot11::more_data(small_uint<1> new_value) {
+    header_.control.more_data = (new_value)? 1 : 0;
```

Comment:
You should be able to just do `= new_value` as that is already in the range [0, 1].

Reply:
I agree.  I was just keeping it consistent with the other fields like power management above.

Reply:
Oh I didn't realize that was the case. Alright, I'll merge it as it is then.